### PR TITLE
Updated translations (DE, FR, CS, PT) for 1.7.

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: openslides/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1894,245 +1925,262 @@ msgstr "Arrêtez de soumettre de nouvelles motions par les utilisateurs non-empl
...
 
-#: motion/signals.py:76
+#: motion/signals.py:75
+msgid "Activate amendments"
+msgstr "Activer amendements"
+
+#: motion/signals.py:80
+msgctxt "Prefix for the identifier for amendments"
+msgid "A"
+msgstr "un"
```
Comment (normanjaeckel) [inline openslides/locale/fr/LC_MESSAGES/django.po]: I think this translation is not correct. The Fench prefix should not be `un` but `A`, should it?

File: openslides/locale/pt/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1890,245 +1921,262 @@ msgstr "Parar a submissão de novas moções por utilizadores não-staff"
...
 
-#: motion/signals.py:76
+#: motion/signals.py:75
+msgid "Activate amendments"
+msgstr "Activar emendas"
+
+#: motion/signals.py:80
+msgctxt "Prefix for the identifier for amendments"
+msgid "A"
+msgstr "A"
```
Comment (normanjaeckel) [inline openslides/locale/pt/LC_MESSAGES/django.po]: Here the same. It should be `E`.

File: openslides/locale/fr/LC_MESSAGES/django.po (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/locale/fr/LC_MESSAGES/django.po]: right, updated!

File: openslides/locale/pt/LC_MESSAGES/django.po (hunk 2 above)
Comment (emanuelschuetze) [inline openslides/locale/pt/LC_MESSAGES/django.po]: should be done, right?

Comment (emanuelschuetze) [inline openslides/locale/pt/LC_MESSAGES/django.po]: ah, that's PT. right "E"! I'll fix it.

**Comment by emanuelschuetze** [comment]
all done. please merge.

**Comment by ostcar** [comment]
ok

**Pull request merged**